Source/Core/InputCommon/ControlReference/ControlReference.cpp

```diff
@@ -48,6 +47,17 @@ ParseStatus ControlReference::GetParseStatus() const
   return m_parse_status;
 }
 
+std::string ControlReference::GetExpression() const
+{
+  return m_expression;
+}
+
+void ControlReference::SetExpression(std::string expr)
+{
+  m_expression = expr;
```

Comment:
When you do a test on a behavior (for assembly), it's best to directly test that behavior over introducing indirection to perform your test. [https://godbolt.org/g/HxwR59](https://godbolt.org/g/HxwR59) which is what I meant about the inlining.